tests/functional/sign_in.js

```diff
@@ -113,7 +113,7 @@ define([
       var email = TestHelpers.createEmail();
       return fillOutSignIn(this, email, PASSWORD)
         // The error area shows a link to /signup
-        .then(FunctionalHelpers.visibleByQSA('.error a[href="/signup"]'))
+        .then(FunctionalHelpers.visibleByQSAHeight('.error a[href="/signup"]', undefined, 10))
```

Comment:
I searched for those links, but they were not used in any `visibleByQSA` functions. Is that what you intended?